tornado_mysql/converters.py

```diff
@@ -123,7 +123,7 @@ def convert_datetime(obj):
         usecs = '0'
         if '.' in hms:
             hms, usecs = hms.split('.')
-        usecs = float('0.' + usecs) * 1e6
+        usecs = round(float('0.' + usecs) * 1e6)
```

Comment:
I haven't checked yet, but is usecs a whole number? If so, seems like it would be simpler to just use that directly (and/or convert it to int from string)?